xml/dm_registers.xml

```diff
@@ -101,26 +117,40 @@
             0 otherwise.
         </field>
         <field name="confstrptrvalid" bits="4" reset="Preset" access="R">
-            0: \RdmConfstrptrZero--\RdmConfstrptrThree hold information which
+            <value v="0" name="invalid">
+            \RdmConfstrptrZero--\RdmConfstrptrThree hold information which
             is not relevant to the configuration structure.
+            </value>
 
-            1: \RdmConfstrptrZero--\RdmConfstrptrThree hold the address of the
+            <value v="1" name="valid">
+            \RdmConfstrptrZero--\RdmConfstrptrThree hold the address of the
             configuration structure.
+            </value>
         </field>
         <field name="version" bits="3:0" reset="3" access="R">
-            0: There is no Debug Module present.
+            <value v="0" name="not present">
+            There is no Debug Module present.
+            </value>
 
-            1: There is a Debug Module and it conforms to version 0.11 of this
+            <value v="1" name="0.11">
+            There is a Debug Module and it conforms to version 0.11 of this
             specification.
+            </value>
 
-            2: There is a Debug Module and it conforms to version 0.13 of this
+            <value v="2" name="0.13">
+            There is a Debug Module and it conforms to version 0.13 of this
             specification.
+            </value>
 
-            3: There is a Debug Module and it conforms to version 1.0 of this
+            <value v="3" name="1.0">
+            There is a Debug Module and it conforms to version 1.0 of this
             specification.
+            </value>
 
-            15: There is a Debug Module but it does not conform to any
+            <value v="15" name="non-compliant">
```

Comment:
"custom" seems less scary.  Unless the goal is to be scary so people are compliant.

Reply:
Done.